addons/wowl/static/src/types.ts

```diff
@@ -65,6 +65,7 @@ export interface SessionInfo {
 
 export interface Odoo {
   session_info: SessionInfo;
+  csrf_token?: string;
```

Comment:
@aab-odoo Optional because it is not present in the mocking. Maybe it should be required ?